### DISCUSSION TITLE
PR's raised by dependabot

### DISCUSSION THREAD

**Discussion by sushmita-m** [github/india#46] [Satellite India 2021 Q&A] [answered]
We have started to vendor our packages using [my-precious](https://www.npmjs.com/package/my-precious), is there a plan to work on making dependabot PR's cater to these vendored packages?

**Comment by dhruvg20** [comment]
👋🏽    Browsing security vulnerabilities in the [GitHub Advisory Database](https://docs.github.com/en/code-security/supply-chain-security/browsing-security-vulnerabilities-in-the-github-advisory-database) would help review all the dependencies. 

We add vulnerabilities to the GitHub Advisory Database from the following sources:

- The [National Vulnerability Database](https://nvd.nist.gov/)
- A combination of machine learning and human review to detect vulnerabilities in public commits on GitHub
- Security advisories reported on GitHub
- The [npm Security advisories](https://www.npmjs.com/advisories) database

🙇🏽

**Comment by nmeans** [comment, marked as answer]
👋 Hi @sushmita-m! Dependabot supports vendored dependencies for the Bundler (Ruby) and Gomod ecosystems, but not currently for npm via my-precious. We don't have plans to add support for that right now, but I'll be sure and pass along the request to the Dependabot team.
